pkg/gather/gather.go

```diff
@@ -100,7 +100,7 @@ func CollectAndRecordGatherer(
 				gathererName, result.FunctionName, err,
 			)
 
-			errs = append(errs, fmt.Errorf(errStr))
+			errs = append(errs, fmt.Errorf("%s", errStr))
```

Comment:
Couldn't we just use `fmt.Errorf` instead of `fmt.Sprintf` in the first place? I don't see the string being used for anything other than to create the error.

Reply:
Yep. Good point. Thanks.